MANIFEST.in

```diff
@@ -1,2 +1,4 @@
 include LICENSE
 include README.rst
+recursive-include formidable/locale *.mo
+recursive-include formidable/locale *.po
```

Comment:
Nitpick: `recursive-include formidable/locale *.[mp]o` is valid in a `MANIFEST.in` file :wink: 
I also saw the directive `graft` to add everything under a subdirectory.